tests/test_backend.py

```diff
@@ -1,19 +1,19 @@
-from PyRDF import *
+import PyRDF
 import unittest
 
 class SelectionTest(unittest.TestCase):
     def test_local_select(self):
-        use("local")
-        self.assertIsInstance(Proxy.backend, Local)
+        PyRDF.use("local")
+        self.assertIsInstance(PyRDF.Proxy.backend, PyRDF.Local)
```

Comment:
Well, in these cases you can have `from PyRDF import Proxy, Local, Dist` above and access them as `Proxy.backend`. The case of `use` was different because it was a quite generic name totally unqualified.

Reply:
But @etejedor , it'd be repetitive to do `import PyRDF` and then again `from PyRDF import Proxy, Local, Dist` right ?

Reply:
I do not see why, you would be importing a module and then importing some of its submodules.

Reply:
Ok @etejedor . I'll fix this in a moment 👍